src/serialize.h

```diff
@@ -1046,6 +1106,19 @@ inline void SerReadWrite(Stream& s, T& obj, int nType, int nVersion, CSerActionU
 }
 
 
+template<typename Stream, typename T, typename C, typename D>
```

Comment:
An analogous consideration regarding naming can be done for `T`, `C` and `D`

Reply:
Makes sense, `T, C, D` -> `Base, Container, Derived`.